### DISCUSSION TITLE
What kits were used for Nanopore library prep?

### DISCUSSION THREAD

**Discussion by alexey-larionov** [GoekeLab/sg-nex-data#11] [Q&A] [answered]
May I assume that you used  PCS109 for your Direct cDNA samples ? 
I need this information to use correct settings in Pychopper.
Thank you for the dataset.

**Comment by alexey-larionov** [comment]
PS: I meant DCS109 of course :) in my question; or was it DCS 108 ?

**Comment by callumparr** [comment, marked as answer]
If you access the preprint, there is a supplementary table detailing a lot the libraries used including the platform and kit used. I reply just because I was reading through it for our research highlight in our labs meeting today.
